parm/config/config.fv3

```diff
@@ -96,7 +96,7 @@ case ${case_in} in
         export layout_y=8
         export layout_x_gfs=${layout_x_gfs:-8}
         export layout_y_gfs=${layout_y_gfs:-12}
-        export nth_fv3=1
```

Comment:
Is this fine with the cycled mode?

Reply:
This was required for the cycled mode to pass. I know @KateFriedman-NOAA was looking for alternatives, I don't know if she found any.

Reply:
So I ran this branch on Hera for 14 cycles and did not encounter any issues using `nth_fv3=1` for C384. I compared my run with Jessica's and found that my `config.fv3` included the commenting out of the lines that adjust the `WRTTASK` variables that are included in this PR (hers did not) and as a result my run used more tasks compared to hers.

Jessica's gdasfcst that had OOM kills:
```
srun -l --export=ALL -n 368
```
Mine that ran fine:
```
srun -l --export=ALL -n 416
```

Test is here on Hera if anyone wants to take a look:
EXPDIR: /scratch1/NCEPDEV/global/Kate.Friedman/expdir/devcyc384
COMROT: /scratch1/NCEPDEV/stmp4/Kate.Friedman/comrot/devcyc384

Looks like we indeed need to comment out the lines to adjust the `WRTTASK` variables based on `npe_node_max` now.

@JessicaMeixner-NOAA please revert this back to `nth_fv3=1` in the C384 case section. Thanks!

Reply:
Thanks @KateFriedman-NOAA, that's great news !!!! I will get this reverted back

Reply:
Awesome, thanks @JessicaMeixner-NOAA ! :)

Reply:
Reverted!